src/status_im2/contexts/wallet/account/tabs/dapps/style.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.wallet.account.tabs.dapps.style
+  (:require
+    [quo2.foundations.colors :as colors]
+    [status-im2.contexts.wallet.account.style :as common-style]))
+
+(def empty-container-style common-style/empty-container-style)
```

Comment:
Imo we should avoid this pattern, it's not expected to have co dependencies between styles, instead let's encapsulate styles within components and use this approach to share styling across many places. 
e,g make a component `(defn empty-container [{} children] ...  [children])` 👍

Reply:
@J-Son89 fixed it

Reply:
thanks!